lib/glific_web/flows/flow_editor_controller.ex

```diff
@@ -361,7 +362,13 @@ defmodule GlificWeb.Flows.FlowEditorController do
         [%{id: "#{c.id}", name: c.name, type: "contact", extra: c.id} | acc]
       end)
 
-    json(conn, %{results: recipients})
+    groups =
+      Groups.list_groups(%{})
+      |> Enum.reduce([], fn g, acc ->
+        [%{id: "#{g.id}", name: g.label, type: "group", extra: g.id} | acc]
```

Comment:
I had a question here. What is g.id here? does it come as a string or integer? and do you need string interpolation here?